log.txt

```diff
@@ -0,0 +1,40 @@
+[38;5;15m[48;5;15m████████[48;5;0m▖▐[48;5;15m██[48;5;0m▌[38;5;0m████[38;5;15m▐[48;5;15m██[48;5;0m▛[38;5;0m██[38;5;15m[48;5;15m███[38;5;0m[48;5;0m███[38;5;15m▝[48;5;15m████████[38;5;0m[48;5;0m███████████[38;5;214m▜[38;5;15m[48;5;215m▝[48;5;230m▌[38;5;214m[48;5;214m█[38;5;172m[48;5;237m▙[38;5;234m[48;5;234m██████████[38;5;239m▝[38;5;255m[48;5;235m▀[38;5;254m▀[38;5;239m▘[38;5;251m▄[38;5;255m[48;5;243m▟[38;5;223m[48;5;214m▘[38;5;214m██▛[48;5;0m▘[38;5;0m████[0m
+[38;5;15m[48;5;15m██[48;5;0m▌[38;5;0m██[38;5;15m▗[48;5;15m███[48;5;0m▝[48;5;15m███[48;5;0m▖[38;5;0m██[38;5;15m▗[48;5;15m███[48;5;0m▌[38;5;0m██[38;5;15m[48;5;15m███[38;5;0m[48;5;0m███[38;5;15m▄▙[38;5;0m████[38;5;15m[48;5;15m███[48;5;0m▌[38;5;0m████████████[38;5;214m▝[48;5;214m▜█[38;5;136m[48;5;234m▌[38;5;234m█████████[38;5;235m▗[38;5;15m[48;5;179m▝[38;5;223m[48;5;214m▀[38;5;215m▚[38;5;230m▄[38;5;15m[48;5;222m▞[48;5;214m▘[38;5;214m[48;5;0m▛[38;5;0m███████[0m
+[38;5;15m[48;5;15m████████[48;5;0m▌[38;5;0m█[38;5;15m▝[48;5;15m████████[48;5;0m▘[38;5;0m███[38;5;15m[48;5;15m███[38;5;0m[48;5;0m██[38;5;15m▜[48;5;15m████████[48;5;0m▛[38;5;0m█████████████████[38;5;214m▀▜[48;5;94m▙[48;5;236m▄[38;5;172m[48;5;234m▄[38;5;136m▄[38;5;130m▄▄[38;5;172m▄[38;5;214m[48;5;237m▄[38;5;223m[48;5;214m▀[38;5;215m▘[38;5;214m[48;5;0m▀▘[38;5;0m██████████[0m
+ _                  _                              _                   _  _      __         _          
+| |                | |                            | |                 | || |    / _|       | |         
+| | __  ___  _   _ | |__    ___    __ _  _ __   __| |__      __  __ _ | || | __| |_   __ _ | | __  ___ 
+| |/ / / _ \| | | || '_ \  / _ \  / _` || '__| / _` |\ \ /\ / / / _` || || |/ /|  _| / _` || |/ / / _ \
+|   < |  __/| |_| || |_) || (_) || (_| || |   | (_| | \ V  V / | (_| || ||   < | |  | (_| ||   < |  __/
+|_|\_\ \___| \__, ||_.__/  \___/  \__,_||_|    \__,_|  \_/\_/   \__,_||_||_|\_\|_|   \__,_||_|\_\ \___|
+              __/ |                                                                                    
+             |___/                                                                                     
+Installing extension::FileWatcher
+Installing motion::KinematicsConfiguration
+Installing support::configuration::GlobalConfig
+Installing support::configuration::SoccerConfig
+Installing support::configuration::NetworkConfiguration
+Installing support::NUsight
+Installing platform::darwin::HardwareSimulator
+Installing platform::darwin::SensorFilter
+Installing motion::QuinticWalk
+Installing motion::HeadController
+Installing behaviour::Controller
+Installing behaviour::skills::DirectWalkController
+Installing behaviour::strategy::KeyboardWalk
+[?1049h[22;0;0t[1;24r(B[m[4l[?7h[?1h=
```

Comment:
What is this doing here :P

Reply:
I thought I deleted that lol
and this is why I dont have a PR for the static walk bc I want to work on multiple computers and I also want to put test stuff in